### PR TITLE
Update lt.rs

### PR DESCRIPTION
**Pull request by MrVinyla**

### CODE REVIEW

File: src/lang/lt.rs [hunk 1]
```diff
@@ -373,17 +373,17 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
...
+		("Enable Recording Session", "Įgalinti seanso įrašinėjimą"),
+		("Allow recording session", "Leisti seanso įrašinėjimą"),
 		("Enable LAN Discovery", "Įgalinti LAN aptikimą"),
 		("Deny LAN Discovery", "Neleisti LAN aptikimo"),
 		("Write a message", "Rašyti žinutę"),
 		("Prompt", "Užuomina"),
 		("Please wait for confirmation of UAC...", "Palaukite UAC patvirtinimo..."),
 		("elevated_foreground_window_tip", "Dabartinis nuotolinio darbalaukio langas reikalauja didesnių privilegijų, todėl laikinai neįmanoma naudoti pelės ir klaviatūros. Galite paprašyti nuotolinio vartotojo sumažinti dabartinį langą arba spustelėti aukščio mygtuką ryšio valdymo lange. Norint išvengti šios problemos ateityje, rekomenduojama programinę įrangą įdiegti nuotoliniame įrenginyje."),
-		("Disconnected", "Atjungtas"),
+		("Disconnected", "Atsijungęs, seanso laikas:"),
```
Comment (rustdesk) [inline src/lang/lt.rs]: `seanso laikas:`?

**Review by rustdesk — CHANGES_REQUESTED** [review]

**Comment by Mr-Update** [comment]
@MrVinyla Can you please use 8 spaces at the beginning of the line instead of the 2 tabs as specified in template.rs.
At the end of the line, LF is provided instead of CRLF.
This will make pull requests based on lang.py much easier.
Thank you very much.

File: src/lang/lt.rs (hunk 1 above)
Comment (MrVinyla) [reply src/lang/lt.rs]: This translates to "session time", but perhaps needs to be changed in the code, not in the translation file. I will fix it. I'm sorry.

**Comment by MrVinyla** [comment]
> @MrVinyla Can you please use 8 spaces at the beginning of the line instead of the 2 tabs as specified in template.rs. At the end of the line, LF is provided instead of CRLF. This will make pull requests based on lang.py much easier. Thank you very much.

I will try to figure out and fix it. Sorry

**Comment by Mr-Update** [comment]
@MrVinyla You can use Notepad++ to change the end of line from CRLF to LF:
- Edit -> EOL Conversion -> Unix (LF)
- File -> Save

**Pull request merged**